server/base/src/test/java/org/apache/accumulo/server/ServiceEnvironmentImplTest.java

```diff
@@ -44,6 +46,12 @@ public void setUp() {
     serviceEnvironment = new ServiceEnvironmentImpl(srvCtx);
   }
 
+  @After
+  public void verifyMocks() {
+    verify(srvCtx);
+    verify(acfg);
+  }
+
```

Comment:
The verify method is actually a varargs, so you can pass both in one:

```suggestion
  @After
  public void verifyMocks() {
    verify(srvCtx, acfg);
  }

```

Reply:
Fixed